src/plugins/workspace/public/components/workspace_creator/workspace_creator_form.tsx

```diff
@@ -123,11 +123,7 @@ export const WorkspaceCreatorForm = (props: WorkspaceCreatorFormProps) => {
           />
           <EuiSpacer size="m" />
           <div {...generateRightSidebarScrollProps(RightSidebarScrollField.Description)} />
-          <WorkspaceDescriptionField
-            value={formData.description}
-            onChange={setDescription}
-            error={formErrors.name?.message}
-          />
+          <WorkspaceDescriptionField value={formData.description} onChange={setDescription} />
```

Comment:
Does description field has validation?

Reply:
No validation for description field for now.

Reply:
Ok, I thought we have 200 size limit

Reply:
Yeah, the workspace description field will turn to red. And the create workspace button will be disabled when description text over 200 characters. But the "Saved changes" button won't be disabled in the edit mode in workspace detail page. Do we need to add this in current PR?

Reply:
I'm fine with having it in a separate PR